<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="send">
       <attribute name="title">
        <string>聊天</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1">
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>接收</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QTextEdit" name="textEdit_recv"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>发送</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="5" column="0">
            <widget class="QPushButton" name="btn_audioStart">
             <property name="text">
              <string>录音</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="btn_sendText">
             <property name="text">
              <string>发送</string>
             </property>
             <property name="shortcut">
              <string>Ctrl+Return</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="btn_audioStop">
             <property name="text">
              <string>停止</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="4">
            <widget class="QTextEdit" name="textEdit_send"/>
           </item>
           <item row="2" column="3">
            <widget class="QPushButton" name="btn_selectFile">
             <property name="text">
              <string>选择文件</string>
             </property>
            </widget>
           </item>
           <item row="5" column="3">
            <widget class="QPushButton" name="btn_audioSend">
             <property name="text">
              <string>发送语音</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QPushButton" name="btn_audioPlay">
             <property name="toolTip">
              <string>播放接收到的语音</string>
             </property>
             <property name="text">
              <string>播放</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="btn_sendFile">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>发送文件</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QProgressBar" name="progressBar_sendFile">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="3">
            <widget class="QLineEdit" name="lineEdit_sendFile">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="recv">
       <attribute name="title">
        <string>图片</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_recvFile">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>视频</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QGroupBox" name="groupBox_8">
          <property name="title">
           <string>本机</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,1">
           <item>
            <layout class="QHBoxLayout" name="hlayout"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QPushButton" name="btn_cameraOpen">
               <property name="text">
                <string>打开摄像头</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_cameraClose">
               <property name="text">
                <string>关闭摄像头</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_capturePic">
               <property name="text">
                <string>截图</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_cameraOpenVideo">
               <property name="text">
                <string>开始视频模式</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_cameraCloseVideo">
               <property name="text">
                <string>关闭视频模式</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_9">
          <property name="title">
           <string>对方</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLabel" name="label_vedioShow">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="test">
       <attribute name="title">
        <string>测试</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_changeLog">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>IP设置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>设备IP：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEdit_deviceIP">
          <property name="text">
           <string>172.16.93.87</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>主机IP：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEdit_pcIP">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QRadioButton" name="rbt_connect">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>AT 指令</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>执行指令:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>设置指令:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="btn_setupSend">
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>查询指令:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QComboBox" name="comboBox_setup"/>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="btn_exeSend">
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QComboBox" name="comboBox_exe"/>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QComboBox" name="comboBox_query">
          <property name="toolTip">
           <string>shishi</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="btn_querySend">
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="3">
         <widget class="QLineEdit" name="lineEdit_setup">
          <property name="toolTip">
           <string/>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="text">
           <string>900</string>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="btn_clearRecv">
          <property name="toolTip">
           <string>清空接收数据窗口以及接收计数值</string>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>参数</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="checkBox_autoSet01">
          <property name="text">
           <string>自动设置01比</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbtn_ATShowHex">
          <property name="text">
           <string>Hex</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbtn_ATShowAscii">
          <property name="text">
           <string>ASCII</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>链路状态</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_statusLight">
             <property name="toolTip">
              <string>灰色：无法通信
绿色：闪烁，心跳包正常</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="res/res.qrc">:/qss/light_gray.png</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_heartBeatCnt">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_heartBeat">
          <property name="text">
           <string>heartBeat</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>链路测试</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_testFrameNumber">
          <property name="toolTip">
           <string>0, 一直发送，直到停止测试</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="label_sendCnt">
          <property name="text">
           <string>发送计数:0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btn_stopTest">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>如果在停止测试后，接受端还能接收到测试数据。请在config.ini文件中减小frameNumber的值</string>
          </property>
          <property name="text">
           <string>stop test</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="toolTip">
           <string>每帧数据1446字节，7+3个rs帧，830us发完</string>
          </property>
          <property name="text">
           <string>发送测试数据帧个数:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="btn_sendTest">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>start test</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>数据帧间隔时间(ms)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_testFrameInterval">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections/>
</ui>
